src/Core/Domain/CmsPageCategory/ValueObject/CmsPageCategoryId.php

```diff
@@ -33,19 +33,37 @@
  */
 class CmsPageCategoryId
 {
+    /**
+     * ID for the topmost Cms Page category
+     */
+    const ROOT_CMS_PAGE_CATEGORY_ID = 1;
+
     /**
      * @var int
      */
     private $cmsPageCategoryId;
 
+    /**
+     * Creates root Cms Page Category ID
+     *
+     * @return CmsPageCategoryId
+     *
+     * @throws CmsPageCategoryException
+     */
+    public static function createRoot()
```

Comment:
@matks wdyt about this change? I've also moved `ROOT_CMS_PAGE_CATEGORY_ID` const from `CmsPageRootCategorySettings` class to `CmsPageCategoryId` as it's more relevant.

Reply:
The idea is nice but where do you use `createRoot()` ?

Reply:
Nowhere. :smile:

My next question is, is it better to use id as `CmsPageCategoryId::ROOT_CMS_PAGE_CATEGORY_ID` or `CmsPageCategoryId::createRoot()->getValue()`. The second option seems better, as it encourages to depend on VO rather than scalars, wdyt?